### DISCUSSION TITLE
How GitHub stars aligns with npm downloads?

### DISCUSSION THREAD

**Discussion by romeerez** [expressots/expressots#114] [General]
Why [npm shows](https://www.npmjs.com/package/@expressots/cli) 146 weekly downloads, how is this possible?

**Comment by rsaz** [comment]
> Why [npm shows](https://www.npmjs.com/package/@expressots/cli) 146 weekly downloads, how is this possible?

Why wouldn't be? As package publisher we don't any internal insight or reporting tool that would clarify how metrics are being captured. However, rule of thumb, they use number of times a packaged has been downloaded over the past 7 days.

Is somewhat accurate if you use other tools to measure the popularity of the repository such as `https://npm-stat.com/`

Any ways, I hope that had clarified your question. :)

**Reply by romeerez** [reply]
I'm publishing various packages on npm, and 146 downloads per week is an indicator of a lib that no one is using besides only the author, that's why it looks very strange. If you open any lib you know that at least a few people are using it, the downloads will be much higher.

So I'm wondering how is that possible? This repo looks good, nice logo, convincing texts, and great docs, and this is a great job that took a lot of effort. And download numbers are close to zero at the same time.

**Reply by rsaz** [reply]
Make sense, maybe you're right about the numbers. Maybe they really don't represent what is going on with the framework.

If you use different stats from npm stats, GitHub, engagement of ExpressoTS in Asia, the weekly download does not represent what is really happening. Anyways, I will leave it here and let the NPM take care of numbers. My mission is to provide the best backend Nodejs+typescript framework companies and individuals can use to build their projects.

If folks like you are enjoying the tool, please consider helping as a contributor embassador etc. You're more than welcome 🤗
